### DISCUSSION TITLE
Make the new code search bar an actual text input

### DISCUSSION THREAD

**Discussion by antoyo** [community/community#55178] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi.
In order to use Vimium's (or other vim-like web browser add-ons) `gi` shortcut to focus the first text input on the page to go to the search bar, it would be great to make the new code search bar an actual text input without having to click on it.
That is, instead of making it a button, make it a text input.
Thanks.

**Comment by locnide** [comment]
I have the same issue. 

Why use `qbsearch-input` instead of `input` ? Isn’t it bad for accessibility ?
